utils/network.go

```diff
@@ -225,16 +223,18 @@ func setupRoutes(hostVeth netlink.Link, result *current.Result) error {
 				// exactly what we are intending to program.
 				// If the route we want is already there then most likely it's programmed by Felix, so we ignore it,
 				// and we return an error if none of the routes match the route we're trying to program.
+				log.Debugf("Constructed route: %v , scope: %v \n", route, route.Scope)
```

Comment:
These debugs all have `\n` at the end; is that intentional?  Also, should we be using `log.WithField()` and friends?